source/content/guides/drupal-9/drupal-9-hosted-createempty-md/11-change-upstream.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Migrate a Site That Was Created with an Empty Upstream to Drupal 9
+subtitle: Change Upstreams
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-createempty-md/change-upstream
+anchorid: change-upstream
+editpath: drupal-9/drupal-9-hosted-createempty-md/11-change-upstream.md
+---
+
+
+Your Pantheon site is now set up to use the Drupal 9 Integrated Composer upstream. To continue tracking additional changes to the Pantheon upstream, change the upstream your site is tracking with Composer:
+
+```bash{promptUser:user}
+terminus site:upstream:set $SITE drupal-recommended
```

Comment:
This will change but I'm not sure what's the new upstream name

Reply:
left as is